app/Http/Controllers/ProjectController.php

```diff
@@ -2,30 +2,152 @@
 
 namespace App\Http\Controllers;
 
+use Illuminate\Http\Request;
+use App\Models\Project;
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Facades\Input;
+use Illuminate\Support\Facades\Session;
+
+
 class ProjectController extends Controller
 {
     /**
-     * Create new project
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // get all the projects
+        $projects = Project::all();
```

Comment:
For stuff like this, I often just do `@foreach (Project::all() as $project)` in the view. Just cleaner